lib/pure/asynchttpserver.nim

```diff
@@ -162,12 +209,13 @@ proc processRequest(
   # Header: val
   # \n
   request.headers.clear()
-  request.body = ""
   request.hostname.shallowCopy(address)
   assert client != nil
   request.client = client
   when (NimMajor, NimMinor) >= (1, 1):
     request.bodyStream = newFutureStream[string]()
+  else:
```

Comment:
what's this change for?

Reply:
Because request.body is not used anymore after 1.1.